README.md

```diff
@@ -3,15 +3,26 @@
 </p>
 
 <p align="center">
+   <a href="https://github.com/emberjs/ember.js/blob/master/LICENSE"><img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="GitHub license"></a>
+   <a href="https://www.npmjs.com/package/ember-source"><img src="https://img.shields.io/npm/v/ember-source.svg?style=flat" alt="npm version"></a>  
   <a href="http://travis-ci.org/emberjs/ember.js"><img src="https://secure.travis-ci.org/emberjs/ember.js.svg?branch=master" alt="Build Status"></a>
   <a href="https://codeclimate.com/github/emberjs/ember.js"><img src="https://codeclimate.com/github/emberjs/ember.js.svg" alt="Code Climate"></a>
   <a href="https://discord.gg/zT3asNS"><img src="https://img.shields.io/discord/480462759797063690.svg?logo=discord" alt="Discord Community Server"></a>
 </p>
 
-Ember.js is a JavaScript framework that greatly reduces the time, effort and resources needed
-to build any web application. It is focused on making you, the developer, as productive as possible by doing all the common, repetitive, yet essential, tasks involved in most web development projects.
+**Ember** is 
+
+* [**Immortal**](https://en.wikipedia.org/wiki/Ember.js) - bulding amazing apps with bleeding edge technologies 8+ years.
+* [**Stable & Secure**](https://emberjs.com/releases/) - LTS Releases, 30 weeks of security fixes
+* [**Modern**](https://blog.emberjs.com/2019/02/11/coming-soon-in-ember-octane-part-1.html) - Classes, Decorators, optional TypeScript, Generators
```

Comment:
Generally, I like where you're going with this overall 🎉 but I wonder if it would be better to sell ember as it is now and then update it to reflect Octane once Octane is actually released? 

I think there is a lot of benefits to "sell" about Ember now even without Octane 😄

Reply:
I agree. There's also the link for `Binary Templates` which seems very experimental and far from production ready

Reply:
@mansona, @makepanic  np, additions/replaces welcome.
Should we mention declarative templates?
My Imagination pretty poor, it will be great if we can have more ideas in this pr.

Reply:
Maybe we should also link https://emberobserver.com/ to highlight the huge addon ecosystem

Reply:
@mansona done as `Zero config addons` section